api/server/server.go

```diff
@@ -1047,6 +1048,11 @@ func (s *Server) postContainersStart(version version.Version, w http.ResponseWri
 		}
 		return err
 	}
+
+	if devmapper.WarnOnLoopback && devmapper.LoopbackInUse {
+		fmt.Fprintln(w, "Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.")
```

Comment:
I am not a big fan of this either, the problem is our file system team is wants it badly to make sure customers do not accidentally  put this into production, and they feel that the log message will be ignored.  I believe there is another patch that writes this message to the syslog system.